### DISCUSSION TITLE
Is there a command to rename a project?

### DISCUSSION THREAD

**Discussion by nbro** [symfony/symfony#54926] [Q&A]
I tried to search for `rename` in the output of `symfony list` or `symfony --help`, but I couldn't find anything relevant thing there.

So, is there a Symfony CLI command to rename a project? Or any other command, maybe from composer, to rename a project, which might have been created with `symfony new`?

For now, I've just renamed the directory, but maybe the project name affects other things other than the project directory.

**Comment by yceruto** [comment]
> For now, I've just renamed the directory

That's all! Nothing else is necessary.
